shared/src/commonMain/kotlin/com/intive/picover/common/result/ResultContractLauncher.kt

```diff
@@ -0,0 +1,11 @@
+package com.intive.picover.common.result
+
+import androidx.compose.runtime.Composable
+import dev.gitlive.firebase.storage.File
+
+expect class ResultContractLauncher {
+	fun launch()
+}
+
+@Composable
+expect fun rememberTakePictureOrPickImageResultContract(onResult: (File) -> Unit): ResultContractLauncher
```

Comment:
Any suggestions about more concise name are welcome 😄

Reply:
maybe just `rememberImageResultContract` 😄

Reply:
maybe just `rememberImageResultContract` 😄

Reply:
Then it would be too generic I think. Maybe from the feature perspective it's too complex because it opens a picker with possibility to launch a camera or open a gallery 😄